modules/ztp-pre-staging-downloading-artifacts.adoc

```diff
@@ -0,0 +1,416 @@
+    - name: registry.redhat.io/multicluster-engine/assisted-installer-rhel8@sha256:33abd6e21cfdc36dd4337fa6f3c3442d33fc3f976471614dca5b8ef749e7a027
+    - name: quay.io/user/troubleshoot
+  operators:
+    - catalog: registry.redhat.io/redhat/redhat-operator-index:v4.11
+      packages:
+        - name: advanced-cluster-management <4>
+          channels:
+             - name: 'release-2.6'
+             - name: 'release-2.5'
+               minVersion: 2.5.4
+               maxVersion: 2.5.4
+        - name: multicluster-engine <4>
+          channels:
+             - name: 'stable-2.1'
+             - name: 'stable-2.0'
+               minVersion: 2.0.4
+               maxVersion: 2.0.4
+        - name: local-storage-operator <3>
+          channels:
+            - name: 'stable'
+        - name: ptp-operator <3>
+          channels:
+            - name: 'stable'
+        - name: sriov-network-operator <3>
+          channels:
+            - name: 'stable'
+        - name: cluster-logging <3>
+          channels:
+            - name: 'stable'
+    - catalog: registry.redhat.io/redhat/certified-operator-index:v4.11
+      packages:
+        - name: sriov-fec <3>
+          channels:
+            - name: 'stable'
+----
+<1> The platform versions match the versions passed to the tool.
+<2> The CR contains the additional images.
+<3> The CR contains all the specified DU Operators.
+<4> The versions of {rh-rhacm} and MCE Operators match the versions passed to the tool.
+
+. Customize the catalog resource in the CR:
++
+[source,yaml]
+----
+apiVersion: mirror.openshift.io/v1alpha2
+kind: ImageSetConfiguration
+mirror:
+  platform:
+---
+  operators:
+    - catalog: eko4.cloud.lab.eng.bos.redhat.com:8443/redhat/redhat-operator-index:v4.11 <1>
+      packages:
+        - name: advanced-cluster-management
+          channels:
+             - name: 'release-2.6'
+             - name: 'release-2.5'
+               minVersion: 2.5.4
+               maxVersion: 2.5.4
+        - name: multicluster-engine
+          channels:
+             - name: 'stable-2.1'
+             - name: 'stable-2.0'
+               minVersion: 2.0.4
+               maxVersion: 2.0.4
+        - name: local-storage-operator
+          channels:
+            - name: 'stable'
+        - name: ptp-operator
+          channels:
+            - name: 'stable'
+        - name: sriov-network-operator
+          channels:
+            - name: 'stable'
+        - name: cluster-logging
+          channels:
+            - name: 'stable'
+    - catalog: eko4.cloud.lab.eng.bos.redhat.com:8443/redhat/certified-operator-index:v4.11 <1>
+----
+<1> The catalog source is changed from an official Red Hat registry.
+// On line 315 and 341 we have the same field with the same value, equal identation. Is that intentional? If yes, what does each one do?
```

Comment:
It's actually a different value, with line 341 referencing the certified-operator-index catalog, vs the redhat-operator-index catalog on line 315. I think this is intended to be a truncated example in the downloading.md doc, with a `...` to indicate the truncation. The key here being that the hostname for the catalog reference is modified from the previous example. @alosadagrande can you confirm?
So the untruncated example would be:
```
    - catalog: eko4.cloud.lab.eng.bos.redhat.com:8443/redhat/certified-operator-index:v4.11
      packages:
        - name: sriov-fec
          channels:
            - name: 'stable'
```

Reply:
Yes, it is intended. This is referring to a specific use case: disconnected environments. Unlike the ocp-release images, the factory-prestaging-tool can query the disconnected catalog source and pull down the proper images from the disconnected registry instead of the RH registry.

I think this is going to be the usual use case in telco where disconnected environments are usual.